onnxruntime/test/providers/cpu/ml/tree_ensembler_classifier_test.cc

```diff
@@ -151,5 +151,107 @@ TEST(MLOpTest, TreeEnsembleClassifierBinary) {
   test.Run();
 }
 
+TEST(MLOpTest, TreeEnsembleClassifierBinaryBaseValue) {
+  OpTester test("TreeEnsembleClassifier", 1, onnxruntime::kMLDomain);
+
+  std::vector<float> base_values = {-1.202673316001892f, -1.202673316001892f};
```

Comment:
Which scikit-learn tree algorithm generated these base_values (e.g. RandomForest, ExtraTrees)?

Would be helpful to put the scikit-learn script that generated this exact tree, or a few comments about how this test case was generated.

Reply:
Fixed.